src/config.js

```diff
@@ -41,6 +41,7 @@ let config = {
   PUBLISHER_BASE_URL: process.env.PUBLISHER_BASE_URL,
   ECOMMERCE_BASE_URL: process.env.ECOMMERCE_BASE_URL,
   ENVIRONMENT,
+  INSIGHTS_BASE_URL: process.env.INSIGHTS_BASE_URL || null,
```

Comment:
Okay, it took me longer than it should have, but I finally reconciled the difference between what I wanted to do here and how the existing system is expected to work.

The question now, somewhat philosophical, is where to implement the changes; here in `frontend-platform` or back in `frontend-app-learning`?

My thinking for these two config variables, was that they should be provided to all services, but they not be required to be set. In fact, I considered that a "feature not a bug" for now.
- If you don't set `INSIGHTS_BASE_URL`, you just don't get the Insights link.
- If you don't set `STUDIO_BASE_URL`, you just don't get the Studio link.

So here, these values are preferred, not required.
Their absence toggles _off_ the dependent feature.

Insights, particularly, we want to make this toggle-able, since not all community installations run Insights, due to its complexity :(

So since the `ensureConfig` and `ensureDefinedConfig` helpers only check that the values not be `undefined`, we can keep them "empty" by explicitly coalescing `falsey` values to `null` (or empty string, if we're concerned about null-check issues).

So, do we think the default `getConfig` should be able to provide empty defaults (code as is here with the latest commit and the `|| null`)?

Or do we want to draw a line in the sand and leave it up the implementing app (move these changes to a `mergeConfig` call in `frontend-app-learning`)?

```js
mergeConfig({
  'INSIGHTS_BASE_URL': process.env.INSIGHTS_BASE_URL || null,
  // ...
});
```